### PR TITLE
fix(template): Fix jest bugs in react template

### PR DESCRIPTION
**Pull request by FDiskas**
This pull request closes issue # (_put the issue number here_)

## Current state

```npm init hops-app my-hops-app```
```cd my-hops-app```
```npm test```

```ssh
Requires Babel "^7.0.0-0", but was loaded with "6.26.3". If you are sure you have a compatible version of @babel/core, it is likely that something in your build process is loading th
e wrong version. Inspect the stack trace of this error to look for the first entry that doesn't mention "@babel/core" or "babel-core" to see what is calling Babel.
```

## Changes introduced here

How to fix:
`npx babel-upgrade --write --install`
`npm audit fix --force`

> **NOTE** And commit `pakage-lock.json` or `yarn.lock` to git for keeping versions locked and not introduce unexpected errors for end users

## Checklist

- [x] All commit messages adhere to the [appropriate format](https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit) in order to trigger a correct release
- [x] All code is written in untranspiled ECMAScript (ES 2017) and is formatted using [prettier](https://prettier.io)
- [x] Necessary unit tests are added in order to ensure correct behavior
- [x] Documentation has been added

### CODE REVIEW

File: packages/template-react/package.json [hunk 1]
```diff
@@ -25,7 +25,9 @@
     "react-router-dom": "^5.0.0"
   },
   "devDependencies": {
-    "jest": "^23.4.2",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "^24.8.0",
```
Comment (ZauberNerd) [inline packages/template-react/package.json]: I would prefer to remove `babel-jest` here, because we have it already as a dependency in the `jest-preset-hops`:
https://github.com/xing/hops/blob/master/packages/jest-preset/package.json#L31

Also, I would like to keep jest at version 23 for now, because unfortunately we still have some discussion points open, for how we could upgrade to jest 24 without being breaking for our users (see https://github.com/xing/hops/issues/815 and https://github.com/xing/hops/pulls?q=is%3Apr+is%3Aopen+label%3Ablocked)

I will try to raise this topic to our attention and investigate if we can find a solution where we can safely upgrade to jest 24, but until then we should stay at 23 to be on the safe side.

So in a nutshell: It will be enough to just add `babel-core@7.0.0-bridge.0` to the dev dependencies ;)

**Comment by ZauberNerd** [comment]
Hi @FDiskas do you want to continue on this PR or should we take over and implement this ourselves?

**Comment by FDiskas** [comment]
Yes please. I'm gone for holiday

**Comment by ZauberNerd** [comment]
@FDiskas I continued your PR here: https://github.com/xing/hops/pull/887
